apps/webservice/src/app/[workspaceSlug]/_components/DeleteDeployment.tsx

```diff
@@ -31,7 +32,11 @@ export const DeleteDeploymentDialog: React.FC<DeleteDeploymentProps> = ({
   const deleteDeployment = api.deployment.delete.useMutation();
 
   const onDelete = async () => {
-    await deleteDeployment.mutateAsync(id);
+    const [_, error] = await safeAwait(deleteDeployment.mutateAsync(id));
+    if (error != null) {
+      console.log(error);
+      return;
+    }
```

Comment:
you don't need to error check, it auto puts it in console

Reply:
```js
const result = await deleteDeployment.mutateAsync(id).catch(console.error)
```

Even if we wanted to do what you suggested this would be the cleaner way